app/services/LocationService.js

```diff
@@ -449,4 +453,70 @@ export default class LocationServices {
       // nav.navigate('LocationTrackingScreen', {});
     });
   }
+
+  static getCrossedPaths() {
+    return new Promise(resolve => {
+      GetStoreData(CROSSED_PATHS).then(dayBin => {
+        dayBin = JSON.parse(dayBin);
+        if (dayBin !== null && dayBin.reduce((a, b) => a + b, 0) > 0) {
```

Comment:
the check is more clearly expressed as `dayBin.some(exposure => exposure > 0)`

reduce has way more overhead.